workspaces/spectacle/src/spectacle.ts

```diff
@@ -312,12 +312,11 @@ export async function makeSpectacle(opticContext: IOpticContext) {
       method: (parent: endpoints.RequestNodeWrapper) => {
         return Promise.resolve(parent.value.httpMethod);
       },
-      query: async () => {
-        // TODO connect up to rust code
-        return {
-          rootShapeId: 'shape-1',
-          isRemoved: false,
-        };
+      query: async (parent: endpoints.RequestNodeWrapper) => {
+        return process.env.REACT_APP_FF_LEARN_UNDOCUMENTED_QUERY_PARAMETERS ===
```

Comment:
@JaapRood put this behind the same FF for simplicity of toggling - but if we decide we want to release learning separate from the UI, we can change this FF

Reply:
Good call. Hopefully that won't be necessary at all (doesn't look like it) :)